internal/process/defunct_processes.go

```diff
@@ -0,0 +1,90 @@
+package process
+
+import (
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// ProcessFS is the process file system.
+const ProcessFS = "/proc"
+
+// Stat represents status information of a process from /proc/[pid]/stat.
+type Stat struct {
+	// Comm is the command name (usually the executable filename).
+	Comm string
+
+	// State is the state of the process.
+	State string
+}
+
+// DefunctProcesses returns the number of zombie processes in the node.
+func DefunctProcesses() (defunctCount uint, err error) {
```

Comment:
nit: can we name this err retErr or something? we have hit weird scoping issues in the past when we use the same name declared in the return values as we use throughout the function

Reply:
Sure, I'll do so.